tests/Validation/ValidationUniqueRuleTest.php

```diff
@@ -3,14 +3,15 @@
 namespace Illuminate\Tests\Validation;
 
 use PHPUnit\Framework\TestCase;
+use Illuminate\Database\Eloquent\Model;
 
 class ValidationUniqueRuleTest extends TestCase
 {
     public function testItCorrectlyFormatsAStringVersionOfTheRule()
     {
         $rule = new \Illuminate\Validation\Rules\Unique('table');
         $rule->where('foo', 'bar');
-        $this->assertEquals('unique:table,NULL,NULL,id,foo,bar', (string) $rule);
+        $this->assertEquals('unique:table,NULL,NULL,NULL,foo,bar', (string) $rule);
```

Comment:
It makes me uncomfortable having to change an old test to add this feature.

Reply:
I think we don't need the default value for the idColumn that's why i had to change the test here, because we really don't need the 'id' since the ignore is NULL. but I can revert this change and still the feature will pass.

Reply:
OK please don't change that test.

Reply:
Done, Sorry for that 👍